tests/testthat/test-loading_workbook.R

```diff
@@ -258,10 +258,10 @@ test_that("test headerFooter", {
   # Load workbook and save again
   wb2 <- wb_load(tmp1)
 
-  expect_equal(wb$worksheets[[1]]$headerFooter,
+  expect_equal(lapply(wb$worksheets[[1]]$headerFooter, unlist),
```

Comment:
I know this is quite ugly, whoever designed these nested lists in the first place gets no cookie today ...